.php_cs.dist

```diff
@@ -1,15 +1,27 @@
 <?php
 
 $finder = PhpCsFixer\Finder::create()
-    ->in(__DIR__)
+    ->in([__DIR__.'/src', __DIR__.'/tests'])
     ->exclude('var')
 ;
 
 return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setCacheFile(__DIR__.'/.php_cs.cache')
     ->setRules([
         '@Symfony' => true,
         'array_syntax' => ['syntax' => 'short'],
         'declare_strict_types' => true,
+        'braces'                                => ['allow_single_line_closure' => true],
+        'native_constant_invocation'            => true,
+        'native_function_invocation'            => [
```

Comment:
why not using `@Symfony:risky` instead of all those rules ?

Reply:
Not all the rules are contained inside Symfony:risky as far as i've checked, and Symfony:risky contains way more rules that the ones i've added;